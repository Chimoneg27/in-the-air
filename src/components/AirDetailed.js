/* eslint-disable import/no-extraneous-dependencies */
import React from 'react';
import PropTypes from 'prop-types';

const AirDetailed = ({ dataname, data }) => {
  const parsedName = React.createElement('div', { dangerouslySetInnerHTML: { __html: dataname } });
  return (
    <div className="data-component-wrap flex">
      <div className="data-component-name">
        {parsedName}
      </div>
      <div className="data-component">
        {data}
      </div>
    </div>
  );
};

AirDetailed.propTypes = {
  dataname: PropTypes.string.isRequired,
  data: PropTypes.string.isRequired,
};

export default AirDetailed;
