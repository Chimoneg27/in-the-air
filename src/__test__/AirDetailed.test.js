import React from 'react';
import { render } from '@testing-library/react';
import AirDetailed from '../components/AirDetailed';

test('renders AirDetailed component', () => {
  const dataname = 'SO';
  const data = '2';
  render(<AirDetailed dataname={dataname} data={data} />);
  expect(AirDetailed).toMatchSnapshot();
});
