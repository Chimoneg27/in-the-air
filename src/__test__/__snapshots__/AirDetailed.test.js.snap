// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders AirDetailed component 1`] = `[Function]`;
